twimi-core/src/usecases/login.rs

```diff
@@ -60,3 +62,12 @@ pub struct LoginOutput {
     pub user: User,
     pub access_token: String,
 }
+
+impl From<(User, String)> for LoginOutput {
+    fn from(value: (User, String)) -> Self {
```

Comment:
ここでパターンマッチングすることもできます

```suggestion
    fn from((user, access_token): (User, String)) -> Self {
```

Reply:
初めて知りました！
便利ですね、これでスッキリです。